docs/reference/query-dsl/boosting-query.asciidoc

```diff
@@ -1,36 +1,76 @@
 [[query-dsl-boosting-query]]
 === Boosting Query
 
-The `boosting` query can be used to effectively demote results that
-match a given query. Unlike the "NOT" clause in bool query, this still
-selects documents that contain undesirable terms, but reduces their
-overall score.
+Returns documents matching a `positive` query while reducing the
+<<query-filter-context, relevance score>> of documents that also match a
+`negative` query.
 
-It accepts a `positive` query and a `negative` query.
-Only documents that match the `positive` query will be included
-in the results list, but documents that also match the `negative` query
-will be downgraded by multiplying the original `_score` of the document
-with the `negative_boost`.
+You can use the `boosting` query to demote certain documents without
+excluding them from the search results.
+
+[[boosting-query-ex-request]]
+==== Example request
 
 [source,js]
---------------------------------------------------
+----
 GET /_search
 {
     "query": {
         "boosting" : {
             "positive" : {
                 "term" : {
-                    "field1" : "value1"
+                    "text" : "apple"
                 }
             },
             "negative" : {
                  "term" : {
-                     "field2" : "value2"
+                     "text" : "pie tart fruit crumble tree"
                 }
             },
-            "negative_boost" : 0.2
+            "negative_boost" : 0.5,
+            "boost" : 1.0 
```

Comment:
Is this the same "boost" factor that we have with every query? In that case I wonder if it complicates the discussion at this point since the "negative_boost" is the thing specific for this query. I was under the impression that we document the "global" boost factor of each query clause elsewhere. Maybe worth reconsidering it here.

Reply:
Thanks again for the review @cbuescher.

This is an excellent point. I removed documentation for the generic `boost` parameter with the `boosting` query with 2f82523.

I've not yet started it yet, but I am planning to document the `boost` parameter with its own page with elastic/elasticsearch#40977.